content/technical_challenges.tex

```diff
@@ -71,36 +71,136 @@ \subsection{Visual Referee Challenge}
 
     \subsubsection{Challenge Goal}
 
-        For the moment robots receive the decision made by the head referee either from a short whistle or a GC message. To improve the perception of the robot and to listen more to the head referee this challenge introduces visual referee signs.
+        In the current SPL rules, the only time that a robot is required to listen directly to the human referees is for the kick-off whistle. Otherwise, all human referee decisions are communicated to the robots via electronic GameController messages. In moving towards the 2050 RoboCup goal, robots will need to directly interpret referee calls and signals (such as whistles, spoken calls and hand signals), rather than receive information from an external electronic source.
+        
+        This technical challenge tests a robot's ability to identify three categories of hand signals:
+        \begin{enumerate}
+            \item Static hand signals with one hand.
+            \item Static hand signals with two hands.
+            \item Dynamic (motion) hand signals with a one or two hands.
+        \end{enumerate}
+        
+        The intent of this challenge is to choose a \emph{subset} of potential referee calls in SPL games and test ability of a robot to recognise different types of hand signals in preparation for adoption in RoboCup games, rather than compile a complete set of all referee signals.
 
     \subsubsection{Challenge setup}
 
-        One robot of the challenged team is placed in the center circle facing the head referee standing on the T-junction opposite to the GC. The referee wears red gloves. The robot watching the referee and listening for the whistle. The general procedure is as follows:
+        One robot of the challenged team is placed in the center circle facing the head referee, standing upright (stiffened) with both hands by its side.
+        %Two robots (from one team) are placed on the field, with one robot on the centre circle kick-off position  for a receiving team and one on the team's penalty spot. Both robots should be facing the kick-off spot.
+        %(Note that it is possible to run the challenge up to two teams at the same time. The referee should take care to balance the direction of hand signals in this case).
+        \todo{TW: Multiple robots could be used for this challenge, and the angle of the robot to the referee could make the challenge more difficult.}
+        The head referee must stand on the T-junction of the center field line, opposite to the GC. The referee must wear a ``black-and-white'' stripped referee jersey (if available) and must  wear red gloves. The purpose of this clothing is to clearly distinguish the referee and their hands from background people. 
```

Comment:
Instead of color can't you use the phrase left and right team out of the perspective of the head ref?

Reply:
I have though about this. In our normal games we refer to colours of the teams, not directions of play. SO my goal here was to keep some parity for this challenge with games rules.

Reply:
Okay